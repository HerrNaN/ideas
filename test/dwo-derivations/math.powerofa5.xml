<request service="derivation" exerciseid="math.powerofa" encoding="string">
  <state>
    <expr>
      (a^(-2))^3
    </expr>
  </state>
</request>
