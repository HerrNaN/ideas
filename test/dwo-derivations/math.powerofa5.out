<reply result="ok" version="0.5.11">
  <list>
    <elem ruleid="mul exponents">
      <expr>
        a^(-6)
      </expr>
    </elem>
  </list>
</reply>

